tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -1,15 +1,28 @@
 \subsection{Graph Theoretical Approach}
+% TODO: Generalise notation in entire section 
```

Comment:
A big general `% TODO` is fine but also include specific ones at each of the lines I pointed out (and any others). It means that work (me reviewing and finding things) doesn't have to be duplicated.